src/settings-ui/Settings.UI/Controls/SettingsPageControl/SettingsPageControl.xaml

```diff
@@ -56,12 +59,17 @@
                     </Border>
 
                     <StackPanel Grid.Column="1">
-                        <TextBlock x:Name="AboutDescription"
-                                   Text="{x:Bind ModuleDescription}"
-                                   Foreground="{ThemeResource TextFillColorSecondaryBrush}"
-                                   TextWrapping="Wrap"/>
+                        <TextBlock
+                            x:Name="AboutDescription"
+                            Foreground="{ThemeResource TextFillColorSecondaryBrush}"
+                            Text="{x:Bind ModuleDescription}"
+                            TextWrapping="Wrap" />
 
-                        <ItemsControl x:Name="PrimaryLinksControl" ItemsSource="{x:Bind PrimaryLinks}" IsTabStop="False" Margin="0,8,0,0">
+                        <ItemsControl
+                            x:Name="PrimaryLinksControl"
+                            Margin="0,8,0,0"
+                            IsTabStop="False"
+                            ItemsSource="{x:Bind PrimaryLinks}">
                             <ItemsControl.ItemTemplate>
                                 <DataTemplate x:DataType="local:PageLink">
                                     <HyperlinkButton NavigateUri="{x:Bind Link}" Style="{StaticResource TextButtonStyle}">
```

Comment:
Is there a reason the TextBlock is needed in place of `Content={x:Bind Text}"` ?

Reply:
Not seeing the benefit of swapping out a TextBlock for a ContentPresenter since the description property is of type string?

Reply:
Not a big deal, just wondering; why use two controls when you can use one?